main.js

```diff
@@ -1,10 +1,123 @@
+//Functions to run at start of webpage
+makeRows()
+
 var scheduledDay = document.querySelector("#scheduled-day");
 var week = document.querySelector(".week");
 week.addEventListener('click', changeDay)
 
 function changeDay(event){
     if(event.target.className !== "week"){
         scheduledDay.innerHTML = "Scheduled Events for " + event.target.textContent;
+        var dayToLowerCase = event.target.innerHTML.toLowerCase()
+
+        switchDay(dayToLowerCase)
+        applyDataModelToDOM(dayToLowerCase)
+    }
+}
+
+var button = document.querySelector("button");
+button.addEventListener('click', displayModal);
+var submit = document.querySelector("#submit");
+submit.addEventListener('click', function (){
+    displayModal()
+    myFormData()
+})
+
+var showModal = false;
+function displayModal(){
+    var modal = document.querySelector(".modal");
+    var transparentBG = document.querySelector(".transparentBG");
+    if(showModal === false){
+        modal.classList.remove("hidden");
+        transparentBG.classList.remove("hidden");
+        showModal = true;
+    } else {
+        modal.classList.add("hidden");
+        transparentBG.classList.add("hidden");
+        showModal = false;
+    }
+}
+
+var descCount = 0;
+var submitText = document.querySelector("#entry");
+var description = document.querySelectorAll(".description");
+var time = document.querySelectorAll(".time");
+var dayOption = document.querySelector("#day");
+var timeOption = document.querySelector("#time")
+
+function myFormData(){
+  var daySelected = dayOption.value;
+  var timeSelected = timeOption.selectedOptions[0].innerHTML;
+  var desc = submitText.value;
+    if(desc === "" || day === 0 || time === 0){
```

Comment:
These are all falsy values right?
`if(!desc || !day || !time)` would be better.